examples/php-generate-json-serializable-preset/__snapshots__/index.spec.ts.snap

```diff
@@ -0,0 +1,28 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Should be able to render PHP and should log expected output to console 1`] = `
+Array [
+  "<?php
+declare(strict_types=1);
+
+namespace Asyncapi;
+
+
+use JsonSerializable;
+final class Root implements JsonSerializable
```

Comment:
```suggestion
declare(strict_types=1);

namespace Asyncapi;

final class Root implements \JsonSerializable
```

How about this outcome?

Reply:
I changed this in 048c9e17d7e0431f032f707e0810dc93ffd4b3c5